package.json

```diff
@@ -4,16 +4,29 @@
   "description": "Make a fetch request from a HAR definition",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "module": "dist/index.mjs",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./types": {
+      "import": "./dist/types.mjs",
+      "require": "./dist/types.js"
+    },
+    "./package.json": "./package.json"
+  },
   "engines": {
     "node": ">=18"
   },
   "scripts": {
-    "build": "tsc",
-    "lint": "eslint . --ext .js,.ts",
-    "prebuild": "rm -rf dist/",
+    "build": "tsup",
+    "lint": "npm run lint:types && npm run lint:ts && npm run prettier",
+    "lint:ts": "eslint . --ext .js,.ts",
+    "lint:types": "tsc",
     "prepack": "npm run build",
-    "pretest": "npm run lint",
-    "prettier": "prettier --list-different --write \"./**/**.{js,ts}\"",
+    "prettier": "prettier --check --write .",
```

Comment:
Do we need to repeat this script if it's performing the same function?

Reply:
oh whoops nice catch, this is a typo — supposed to be this: 97e982d8472ee345a790272183ee89228b937d76